astropy/units/tests/test_structured.py

```diff
@@ -558,7 +561,7 @@ def test_setitem(self):
         assert np.all(q_pv_t['pv'] == (1., 0.5) * self.pv_unit)
 
 
-class TestStructuredQuantityFunctions(StructuredTestBaseWithUnits):
+class TestStructuredQuantityFunctions(StructuredTestBaseWithUnits, metaclass=CoverageMeta):
```

Comment:
I think the metaclass is no longer needed.

Reply:
Agreed. Removing.

Reply:
✅